app/pipelines/routes.py

```diff
@@ -140,24 +140,39 @@ def pipeline_search():
     # else take the first one and set logo
     # TODO right now, this handles CBRAIN and one other platform
 
+    #Mandana, issue 378
+    with open(os.path.join(cache_dir, "cbrain-conp-pipeline.json"), "r") as f:
+        zenodoUrls = json.load(f)
+            
     for element in elements_on_page:
         element["platforms"] = [{} for x in range(0, 1)]
         element["platforms"][0]["img"] = url_for(
             'static', filename="img/run_on_cbrain_gray.png")
         element["platforms"][0]["uri"] = ""
-
+'''
         if "onlineplatformurls" in element:
             # Check CBRAIN
             for url in element["onlineplatformurls"]:
                 if url.startswith('https://portal.cbrain.mcgill.ca'):
+                #if element["id"] in zenodoUrls.keys():
                     element["platforms"][0]["img"] = url_for(
                         'static', filename="img/run_on_cbrain_green.png")
                     element["platforms"][0]["uri"] = url
                 else:
                     platform_dict = {"img": url_for('static', filename="img/globe-solid-green.png"),
                                      "uri": url}
                     element["platforms"].append(platform_dict)
-
+'''
```

Comment:
the section between `'''` line 152 and `'''` line 165 should be removed since this is commented code

Reply:
thanks Cecile, I resolved them all, I hope it works now